pkg/server/structlogging/structlogging_test: TestHotRangesLoggingJobExitProcedure failed

pkg/server/structlogging/structlogging_test.TestHotRangesLoggingJobExitProcedure [failed](https://mesolite.cluster.engflow.com/invocations/default/54d41ee4-c127-442f-adfe-677c58edf240?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyL3N0cnVjdGxvZ2dpbmc6c3RydWN0bG9nZ2luZ190ZXN0) on master @ [e074bcf710e6de238a82058d3fdc802cb819a06c](https://github.com/cockroachdb/cockroach/commits/e074bcf710e6de238a82058d3fdc802cb819a06c):


```
=== RUN   TestHotRangesLoggingJobExitProcedure
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestHotRangesLoggingJobExitProcedure1037732617
    test_log_scope.go:82: use -show-logs to present logs inline
    hot_ranges_log_job_test.go:42: condition failed to evaluate within 45s: from hot_ranges_log_job_test.go:53: app job status is succeeded, not running
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestHotRangesLoggingJobExitProcedure1037732617
--- FAIL: TestHotRangesLoggingJobExitProcedure (45.97s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/obs-prs
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestHotRangesLoggingJobExitProcedure.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-52408

pkg/server/structlogging/structlogging_test.TestHotRangesLoggingJobExitProcedure [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20147200?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20147200?buildTab=artifacts#/) on master @ [e074bcf710e6de238a82058d3fdc802cb819a06c](https://github.com/cockroachdb/cockroach/commits/e074bcf710e6de238a82058d3fdc802cb819a06c):


```
=== RUN   TestHotRangesLoggingJobExitProcedure
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/c6127e20ddcfbeda2eae8d7e7953ee47/logTestHotRangesLoggingJobExitProcedure533350278
    test_log_scope.go:82: use -show-logs to present logs inline
    hot_ranges_log_job_test.go:42: condition failed to evaluate within 45s: from hot_ranges_log_job_test.go:53: app job status is succeeded, not running
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c6127e20ddcfbeda2eae8d7e7953ee47/logTestHotRangesLoggingJobExitProcedure533350278
--- FAIL: TestHotRangesLoggingJobExitProcedure (46.95s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestHotRangesLoggingJobExitProcedure.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/server/structlogging/structlogging_test.TestHotRangesLoggingJobExitProcedure [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20150450?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20150450?buildTab=artifacts#/) on master @ [9b7bfc295960bfb86b32db9d2b29188d6acfabf2](https://github.com/cockroachdb/cockroach/commits/9b7bfc295960bfb86b32db9d2b29188d6acfabf2):


```
=== RUN   TestHotRangesLoggingJobExitProcedure
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/c6127e20ddcfbeda2eae8d7e7953ee47/logTestHotRangesLoggingJobExitProcedure1389695823
    test_log_scope.go:82: use -show-logs to present logs inline
    hot_ranges_log_job_test.go:42: condition failed to evaluate within 45s: from hot_ranges_log_job_test.go:53: app job status is succeeded, not running
    panic.go:635: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/c6127e20ddcfbeda2eae8d7e7953ee47/logTestHotRangesLoggingJobExitProcedure1389695823
--- FAIL: TestHotRangesLoggingJobExitProcedure (46.21s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestHotRangesLoggingJobExitProcedure.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

pkg/server/structlogging/structlogging_test.TestHotRangesLoggingJobExitProcedure [failed](https://mesolite.cluster.engflow.com/invocations/default/2c91202f-1de0-453a-aa5d-9ba71ba3300f?testReportRun=1&testReportShard=1&testReportAttempt=1#targets-Ly9wa2cvc2VydmVyL3N0cnVjdGxvZ2dpbmc6c3RydWN0bG9nZ2luZ190ZXN0) on master @ [9b7bfc295960bfb86b32db9d2b29188d6acfabf2](https://github.com/cockroachdb/cockroach/commits/9b7bfc295960bfb86b32db9d2b29188d6acfabf2):


```
=== RUN   TestHotRangesLoggingJobExitProcedure
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestHotRangesLoggingJobExitProcedure936243461
    test_log_scope.go:82: use -show-logs to present logs inline
    hot_ranges_log_job_test.go:42: condition failed to evaluate within 45s: from hot_ranges_log_job_test.go:53: app job status is succeeded, not running
    panic.go:635: -- test log scope end --
test logs left over in: outputs.zip/logTestHotRangesLoggingJobExitProcedure936243461
--- FAIL: TestHotRangesLoggingJobExitProcedure (46.25s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=1</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestHotRangesLoggingJobExitProcedure.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/server/structlogging/hot_ranges_log_job_test.go
+++ b/pkg/server/structlogging/hot_ranges_log_job_test.go
@@ -27,6 +27,7 @@ func TestHotRangesLoggingJobExitProcedure(t *testing.T) {
 	defer log.Scope(t).Close(t)
 	skip.UnderStress(t)
 	skip.UnderRace(t)
+	skip.WithIssue(t, 149977)
 
 	ctx := context.Background()
 	ts := serverutils.StartServerOnly(t, base.TestServerArgs{

